charts/kaskada-canary/README.md

```diff
@@ -15,50 +15,3 @@ Add the following repo to use the chart:
 ```console
 helm repo add kaskada https://kaskada-ai.github.io/kaskada/
 ```
-
-## Values
```

Comment:
too hard to keep this `values` documentation up-to-date in two places.  instead documented on the file itself.